src/parser/jobs/gnb/modules/NoMercy.tsx

```diff
@@ -77,10 +78,11 @@ export class NoMercy extends BuffWindow {
 		this.addEvaluator(new ExpectedGcdCountEvaluator({
 			expectedGcds: EXPECTED_USES.GCD,
 			globalCooldown: this.globalCooldown,
+			hasStacks: false,
 			suggestionIcon,
 			suggestionContent: <Trans id="gnb.nomercy.suggestions.gcds.content">
-				Try to land 9 GCDs during every <ActionLink action="NO_MERCY" /> window. A 20 second duration is sufficient
-					to comfortably fit 9 GCDs with full uptime if you wait until the last one-third of your GCD timer to activate it.
+				Try to land {EXPECTED_USES.GCD} GCDs ({EXPECTED_USES.GCD_SLOW} without gearing skill speed) during every <ActionLink action="NO_MERCY" /> window.
```

Comment:
Should this copy specify the GCD threshold that's too slow to fit 9 GCDs?  Seems ambiguous as written.

Reply:
Something like this?
"Try to land {EXPECTED_USES.GCD} weaponskills ({EXPECTED_USES.GCD_SLOW} weaponskills at {GCD_SLOW} GCD or slower) during every <ActionLink action="NO_MERCY" /> window."

Or without the variable names,
"Try to land 9 weaponskills (8 weaponskills at 2.47 GCD or slower) during every No Mercy window."

Reply:
That seems like an improvement to me.